### PR TITLE
Spw 1047 gulp

### PR DESCRIPTION
**Pull request by dejakob**
http://master.showpad.dev:8080/job/AngularTags-SPW-1047-gulp/10
http://team.showpad.com:8008/browse/SPW-1047

### CODE REVIEW

File: gulp/build.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+    gulp.src(paths.style.bootstrap.src)
+        .pipe(sass())
+        .pipe(concat('ng-tags-input.bootstrap.css'))
+            .pipe(gulp.dest(paths.dist))
+            .pipe(minify())
+            .pipe(rename({suffix: '.min'}))
+            .pipe(gulp.dest(paths.dist));
+
+    done();
```
Comment (klaascuvelier) [inline gulp/build.js]: this makes no sense.
The other 2 are async, you are calling done here way too soon.

Comment (klaascuvelier) [inline gulp/build.js]: Quickfix:
Make 2 'sub-style-tasks': `tagsStyleTask` and `bootstrapStyleTask` (add `return` in those functions!), and then is `style` just executing those 2 subtasks.

File: gulp/serve.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+var paths   = require('./paths');
+
+var gulp    = require('gulp');
+
+gulp.task('watch', function () {
+    gulp.watch(paths.style.src, ['styles']);
+    gulp.watch(paths.style.bootstrap.src, ['styles']);
+    gulp.watch(paths.scripts.src, ['scripts']);
```
Comment (klaascuvelier) [inline gulp/serve.js]: No templates watching?

File: package.json [hunk 3]
```diff
@@ -1,57 +1,52 @@
...
+    "url": "https://github.com/showpad/angular-tags/issues"
+  },
+  "keywords": [
+    "angular",
+    "tags",
+    "tags-input"
+  ],
+  "main": "build/ng-tags-input.min.js",
+  "dependencies": {
+    "gulp": "^3.9.0",
```
Comment (klaascuvelier) [inline package.json]: use specific versoins for all dependencies, no caret versions or what so ever.

**Comment by klaascuvelier** [comment]
Amount of tests went down quite a lot, how did this happen? 
http://master.showpad.dev:8080/job/AngularTags-SPW-1047-gulp/

File: gulp/serve.js (hunk 2 above)
Comment (dejakob) [inline gulp/serve.js]: Yeah, indeed, forgot to add that

**Comment by dejakob** [comment]
Simple, I made a couple of mistakes in the configuration at the beginning. Because of that, it was counted twice ;)

**Pull request merged**